src/components/ItemBox.js

```diff
@@ -6,16 +6,16 @@ function ItemBox(props) {
   const shelf_divs = props.items;
 
   let [count, setCount] = useState(200);
-  //Displays an array of itmes from a call to the
+
   const displayItems = shelf_divs.map((division, index) => (
     <li key={index}>
       <div className="itemBox">
-        {/* <div className="itemQuantBox"> */}
         <button className="infoButton">i</button>
-        <textarea class="scrollableTextBox" name="itemName">
-          {division.shelf_div}
```

Comment:
This line was throwing an error. I looked it up, changed it, and displays correctly.